Shared/Samples/Browse OGC API feature service/BrowseOGCAPIFeatureServiceView.swift

```diff
@@ -0,0 +1,260 @@
+                                if let selection = model.selectedInfo {
+                                    do {
+                                        try await model.displayLayer(with: selection)
+                                        if let extent = model.completeExtent {
+                                            await mapProxy.setViewpointGeometry(
+                                                extent,
+                                                padding: 100
+                                            )
+                                        }
+                                    } catch {
+                                        self.error = error
+                                    }
+                                }
+                            }
+                            .pickerStyle(.automatic)
+                        }
+                    }
+                }
+                .alert("Load OGC API feature service", isPresented: $presentAlert, actions: {
+                    // Alert is set with a default url for the OGC API, the user can update the url.
+                    TextField("URL:", text: $userInput)
+                    Button("Load", action: {
+                        presentAlert = false
+                        isLoading = true
+                        Task {
+                            do {
+                                try await model.loadOGCFeatureData(url: URL(string: userInput))
+                                // This selects the first layer in the layer name list. This is
+                                // needed so that the layer selection picker is set in the toolbar.
+                                selection = model.layerNames.first ?? ""
+                                if let extent = model.completeExtent {
+                                    await mapProxy.setViewpointGeometry(
+                                        extent,
+                                        padding: 100
+                                    )
+                                }
+                            } catch {
+                                self.error = error
+                            }
+                        }
+                    })
+                    Button("Cancel", role: .cancel, action: {
+                        presentAlert = false
+                    })
+                }, message: {
+                    Text("Please provide a URL to an OGC API feature service.")
+                })
+                .errorAlert(presentingError: $error)
+        }
+    }
+}
+
+private extension BrowseOGCAPIFeatureServiceView {
+    @MainActor
+    class Model: ObservableObject {
+        let map: Map = {
+            let map = Map(basemapStyle: .arcGISTopographic)
+            map.initialViewpoint = Viewpoint(
+                center: Point(
+                    latitude: 32.62,
+                    longitude: 36.10
+                ),
+                scale: 200000
```

Comment:
```suggestion
                scale: 200_000
```

or 2e5 for simplicity